examples/fft_physical_x/fft_c2c_x.f90

```diff
@@ -143,11 +143,18 @@ program fft_c2c_x
       end do
    end do
    !$acc end loop
-   call MPI_ALLREDUCE(error, err_all, 1, real_type, MPI_SUM, MPI_COMM_WORLD, ierror)
-   err_all = err_all / (real(nx, mytype) * real(ny, mytype) * real(nz, mytype))
+   call MPI_ALLREDUCE(MPI_IN_PLACE, error, 1, real_type, MPI_SUM, MPI_COMM_WORLD, ierror)
+   error = error / (real(nx, mytype) * real(ny, mytype) * real(nz, mytype))
+
+   ! Abort if the error is too high
+   ! A large enough value is needed for the generic backend
+   if (error > epsilon(error) * 50 * ntest) then
```

Comment:
Does error accumulate across all tests?

Reply:
No i could just multiply by 1000 and will be fine. Issue is mainly generic FFT wehre error is e-14 instead of e-16